packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -350,5 +352,11 @@ public void onReattachedToActivityForConfigChanges(ActivityPluginBinding binding
   public void onDetachedFromActivity() {
     lifecycle = null;
     channel.setMethodCallHandler(null);
+    activity = null;
+  }
+
+  @VisibleForTesting
+  /*package*/ final Activity getActivity() {
```

Comment:
Please don't check in commented-out code.

Reply:
> Please don't check in commented-out code.

Thank you for replying.

This note was added intentionally. Package level protection is the default, but adding the note there will be more clear.

Reply:
> > Please don't check in commented-out code.
> 
> This note was added intentionally.

I understand that you did it intentionally, but we don't check in things like this.

> Package level protection is the default, but adding the note there will be more clear.

We don't comment standard language behavior.

Reply:
> > > Please don't check in commented-out code.
> > 
> > 
> > This note was added intentionally.
> 
> I understand that you did it intentionally, but we don't check in things like this.
> 
> > Package level protection is the default, but adding the note there will be more clear.
> 
> We don't comment standard language behavior.

Thanks, I got it now.